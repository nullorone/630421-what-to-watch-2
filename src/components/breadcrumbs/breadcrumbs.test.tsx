import * as React from "react";
import renderer from "react-test-renderer";
import Breadcrumbs from "./breadcrumbs";
import {BrowserRouter} from "react-router-dom";

describe(`Make snapshot`, () => {
  const initProps = {
    breadcrumbs: [
      {
        text: `Keks`,
        link: `https://htmlacademy.ru`
      }
    ],
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <Breadcrumbs {...initProps}/>
          </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
