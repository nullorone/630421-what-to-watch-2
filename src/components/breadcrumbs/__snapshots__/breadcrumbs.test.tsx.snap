// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="https://htmlacademy.ru"
      >
        Keks
      </a>
    </li>
  </ul>
</nav>
`;
